website/content/docs/concepts/security/permissions/grants.mdx

```diff
@@ -0,0 +1,182 @@
+
+- &#32; `*` (all)
+- &#32; `session`
+- &#32; `target`
+- &#32; `user`
+- &#32; `worker`
+
+For a full list of types, see the [Resource Table](/boundary/docs/concepts/security/permissions/resource-table).
+
+## Actions
+
+**Actions** convey the ability to perform some action against a resource or collection.
+You can select which actions you want to apply to a grant.
+You can also use a [wildcard](/boundary/docs/concepts/security/permissions/permission-grant-formats#wildcard-id) to include all valid actions.
+Each grant string can include one or more comma separated actions.
+Some commonly used actions include:
+
+- &#32; `*` (all)
+- &#32; `list`
+- &#32; `create`
+- &#32; `read`
+- &#32; `update`
+- &#32; `delete`
+
+To learn more about actions, see [advanced actions](/boundary/docs/concepts/security/permissions/advanced#advanced-actions).
+For a full list of actions, see the [Resource Table](/boundary/docs/concepts/security/permissions/resource-table).
+
+## Output fields
+
+**Output fields** are top-level fields that are returned in the response. Output fields
+are **optional** and not commonly used.
+
+Examples of valid output fields include:
+
+- &#32; `none`
+- &#32; `id`
+- &#32; `name`
+- &#32; `description`
+
+To learn more about output fields, see [output fields](/boundary/docs/concepts/security/permissions/advanced#output-fields)
+
+## Common grant examples
+
+The following are examples of commonly used grant strings for specific roles and scopes.
+
+**All actions** 
+
+This grant string allows users to perform any actions on any resources in the scope:
+
+- &#32; `id=*;type=*;actions=*`
+
+**View all** 
+
+This grant string allows users to list and read any resources in the scope:
+
+- &#32; `id=*;type=*;actions=read,list`
+
+**View targets only** 
+
+This **project scope** grant string allows users to list and read any targets in the specific project.
+This example is only applicable to project scopes because targets can only exist inside a project:
+
+- &#32; `id=*;type=target;actions=read,list`
+
+**Connect to any target**
+
+This **project scope** grant string allows users to list, read, and connect to any targets in a specific project.
+In addition, these grants allow the user to list, read, and cancel any sessions initiated by the same user.
+This example is only applicable to project scopes, since targets can only exist in a project:
+
+- &#32; `id=*;type=target;actions=list,read,authorize-session`
+- &#32; `id=*;type=session;actions=read:self,cancel:self,list`
+
+**Connect to a specific target** 
+
+This **project scope** grant string allows users to list, read, and connect to a specific target.
+In addition, these grants let users list, read, and cancel any sessions initiated by the same user.
+If you want to include several specific targets, you can duplicate the first grant string using the respective IDs of those targets.
+This example is only applicable to project scopes, since targets can only exist in a project:
```

Comment:
```suggestion
If you want to include several specific targets, you can define the id to be a comma separated list containing the respective IDs.
This example is only applicable to project scopes, since targets can only exist in a project:
```